cypress.config.js

```
@@ -1,6 +1,6 @@
-import { defineConfig } from "cypress";
+const { defineConfig } = require("cypress");
```

ESLint

[error] Definition for rule 'import/no-extraneous-dependencies' was not found.

(import/no-extraneous-dependencies)